src/coreclr/jit/liveness.cpp

```diff
@@ -300,6 +300,19 @@ void Compiler::fgPerNodeLocalVarLiveness(GenTree* tree)
             fgCurMemoryDef |= memoryKindSet(GcHeap, ByrefExposed);
             break;
 
+#ifdef FEATURE_SIMD
+        case GT_SIMD:
+        {
+            GenTreeSIMD* simdNode = tree->AsSIMD();
+            if (simdNode->OperIsMemoryLoad())
```

Comment:
Should this also handle stores to ensure it matches the `case GT_HWINTRINSIC` path below?

Reply:
Ah, we don't have `::OperIsMemoryStore` for `GT_SIMD` 👍

Reply:
Yeah, apparently there aren't any SIMD store instructions (and only one SIMD load instruction)